src/models/PolicyList.ts

```diff
@@ -368,7 +380,7 @@ class PolicyList extends EventEmitter {
                 // Invalid/unknown rule, just skip it.
                 continue;
             }
-            event.unsigned.rule = rule;
+            event[PolicyList.EVENT_RULE_ANNOTATION_KEY] = rule;
```

Comment:
A bit of a scary change. I trust that you have ported all the instances of `unsigned`, right?